src/scss/modules/sidebar.scss

```diff
@@ -53,7 +53,7 @@
 .sidebar__attachments {
   border-bottom: 1px solid $black-90;
   border-top: 1px solid $black-90;
-  padding: 16px 0 24px;
+  padding: 16px 0;
```

Comment:
Not entirely sure why this had the `padding-bottom: 24px` before...

But both the program action sidebar and action sidebar looks good to me with `padding-bottom: 16px`:

![Screenshot 2023-05-03 at 4 07 38 PM](https://user-images.githubusercontent.com/35355575/236050442-f92b10da-8787-4fc0-98ee-edff0c31b01c.png)
![Screenshot 2023-05-03 at 4 10 46 PM](https://user-images.githubusercontent.com/35355575/236050840-4eacdcd6-c323-4d76-aa92-bd58a0d059af.png)